corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -240,9 +255,13 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                 answers = {},
                 model = this.parentModel;
             $fields.each(function (index) {
-                if (this.value !== '') {
-                    answers[model[index].get('id')] = encodeValue(model[index], this.value);
+                if (this.tagName === 'SELECT' && this.multiple) {
+                    var val = $(this).val();
+                }
+                else if (this.value !== '') {
+                    val = this.value;
                 }
+                answers[model[index].get('id')] = encodeValue(model[index], val);
```

Comment:
I think this should only be done if `val` is not empty right?

Reply:
Ah, true. Will update.